packages/devtools_app/lib/src/profiler/cpu_profiler.dart

```diff
@@ -87,6 +97,21 @@ class _CpuProfilerState extends State<CpuProfiler>
               controller: _tabController,
               tabs: CpuProfiler.tabs,
             ),
+            // TODO(kenz): support search for call tree and bottom up tabs as
+            // well. This will require implementing search for tree tables.
+            if (currentTab.key == CpuProfiler.flameChartTab &&
```

Comment:
conditioning UI based on whether a key is present is a bit surprising. Maybe just pass a boolean indicating whether to include a search field and define a SearchField widget so you can match for it in the tests if that is the only reason you have a key.